res/database/Default/database_programmes.xml

```diff
@@ -8917,24 +9063,24 @@ Weary of the twins' constant pranks, his helping-out sister also strongly recomm
 			</children>
 		</programme>
 
-		<programme id="TheRob-b5dc-4663-bd81-Ronnyunddasbyte" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0061418" creator="8751" created_by="TheRob">
+		<programme id="a51df750-cd95-49be-9d16-ffb8e48513bd" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0061418" creator="8751" created_by="TheRob" comment="previous spoof name: 'Ronny und das Byte' - now 'forked' to d64fae37-49c0-4ba6-a6b5-ef52b1f9399a">
 			<title>
 				<!-- Bonnie und Clyde, Krimi, sub romance -->
```

Comment:
In other cases, you removed these comments

Reply:
Ah. For now I kept these in where the genres are mentioned. In other cases the comments were just redundant.